lisa/transformers/kernel_installer.py

```diff
@@ -140,37 +142,17 @@ def _internal_run(self) -> Dict[str, Any]:
             if installer.runbook.source != "linux-image-azure-fde":
                 posix = cast(Posix, node.os)
                 posix.replace_boot_kernel(installed_kernel_version)
-            else:
```

Comment:
Please check with the code author understand if it's redundant.

Reply:
@LiliDeng This CVM logic is no longer required and causing failures as it cannot find .efi files under "/usr/lib/linux/efi/".

Reply:
will you run old distro which needs this logic?

Reply:
No, we only use this with proposed kernel kernels. This logic is not needed on all recent CVM kernels on both focal and jammy.